documentation/modules/managing/con-namespace-deletion-recovery.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="namespace-deletion_{context}"]
+= Cluster recovery from namespace deletion
+
+Recovery from namespace deletion is possible because of the relationship between persistent volumes and namespaces.
+A `PersistentVolume` (PV) is a storage resource that lives outside of a namespace.
+A PV is mounted into a Kafka pod using a `PersistentVolumeClaim` (PVC), which lives inside a namespace.
+
+If a namespace is deleted unintentionally, you can recover it from a PV as long as it is still present.
```

Comment:
This might be my fault because of how I put together the original document. But I think this is a bit out of order. We first say _if namespace is deleteed you can recover_ and only then we say _oh, by the way unless you changed this setting the PVs will be deleted_. I think we need to put this into the right order:
* Explain the policy
* Tell them they need to use Retain for this

Reply:
Rewritten to:

> The reclaim policy for a PV tells a cluster how to act when a namespace is deleted.
> If the reclaim policy is set as:
> 
> *  _Delete_ (default), PVs are deleted when PVCs are deleted with a namespace
> * _Retain_, PVs are not deleted when a namespace is deleted
> 
> To ensure that you can recover from a PV if a namespace is deleted unintentionally, the policy must be reset from _Delete_ to _Retain_.